packages/react-aria-components/src/Collection.tsx

```diff
@@ -9,10 +9,10 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-import {CollectionBase, LinkDOMProps} from '@react-types/shared';
+import {CollectionBase} from '@react-types/shared';
```

Comment:
kind of a nice side effect that link props no longer need to be in here, so there could be collections which don't support links now